inst/include/Rcpp/vector/Matrix.h

```diff
@@ -145,11 +149,14 @@ class Matrix : public Vector<RTYPE, StoragePolicy>, public MatrixBase<RTYPE, tru
     inline Row operator()( int i, internal::NamedPlaceHolder ) {
       return Row( *this, i ) ;
     }
+    inline ConstRow operator()( int i, internal::NamedPlaceHolder ) const {
+      return ConstRow( *this, i ) ;
+    }
     inline Column operator()( internal::NamedPlaceHolder, int i ) {
       return Column( *this, i ) ;
     }
-    inline Column operator()( internal::NamedPlaceHolder, int i ) const {
```

Comment:
This, to my mind, is the only suspicious part.  The function should never have been here and I have changed the return type to be appropriate, but any code depending on it will likely break.